scenarios/group-get-and-stats.har

```diff
@@ -0,0 +1,263 @@
+{
```

Comment:
Ah... I have removed the call to group-stats but forgot to rename the file. The .har file should be fine.